src/rabbit_vm.erl

```diff
@@ -173,11 +175,14 @@ bytes(Words) ->  try
                  end.
 
 interesting_sups() ->
-    [queue_sups(), conn_sups() | interesting_sups0()].
+    [queue_sups(), quorum_sups(), conn_sups() | interesting_sups0()].
 
 queue_sups() ->
     all_vhosts_children(rabbit_amqqueue_sup_sup).
 
+quorum_sups() ->
+    [ra_server_sup].
```

Comment:
This thing implies that all ra processes are quorum queues. Although it's currently true, we should take a note and change that later.
Maybe we can filter the qeueues somehow? I'm not sure if it's possible.